src/main/kotlin/dk/cachet/carp/webservices/protocol/repository/CoreProtocolRepository.kt

```diff
@@ -205,4 +205,21 @@ class CoreProtocolRepository(
         return null
     }
 
+    /**
+     * The [getLatestProtocols] function sorts Protocols conditionally by  getLatestProtocolById()
+     * which results of having the latest updated Protocols.versionTag
+     * converts a [JsonNode] to a [GetLatestProtocolResponseDto].
+     *
+     * @return A [GetLatestProtocolOverviewResponseDto] object containing the ProtocolOverviewDto.
+     */
+
+    suspend fun getLatestProtocols(): List<GetLatestProtocolOverviewResponseDto?> =
```

Comment:
Oh yeah, last time I forgot to mention this, but could you write tests for this?